firefox/manifest.json

```diff
@@ -11,21 +11,19 @@
     }
   },
   "icons": {
-    "16": "dist/img/favicon/favicon-16x16.png",
-    "32": "dist/img/favicon/favicon-32x32.png",
-    "64": "dist/img/favicon/favicon-64x64.png",
-    "96": "dist/img/favicon/favicon-96x96.png",
-    "128": "dist/img/favicon/favicon-128x128.png"
+    "32": "dist/img/rested-logo.svg"
```

Comment:
I had to look this up. It turns out that even though you use SVG and only one file, you still should provide all the different sizes. Could you please provide them like before here, and in 16 and 32 in `default_icon` and `theme_icons`?

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/icons#SVG